main.go

```diff
@@ -138,6 +139,19 @@ func getConnWithRetry(flags *flagStorage) (c *gcsx.Connection, err error) {
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
+func CreateStorageHandle(flags *flagStorage) (storageHandle storage.StorageHandle, err error) {
+	var tokenSrc oauth2.TokenSource
+
+	tokenSrc, err = auth.GetTokenSource(context.Background(), flags.KeyFile, flags.TokenUrl, true)
+	var storageClientConfigToken = storage.StorageClientConfig{TokenSrc: tokenSrc}
+	if err != nil {
+		err = fmt.Errorf("get token source: %w", err)
+		return
+	}
```

Comment:
please add new lines as and when appropriate. Here we have a err check due to which we might exit from the method execution, so its good to add a new line here to distinguish the logic

Reply:
fixed